compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Subst.hs

```diff
@@ -182,6 +182,10 @@ applySubstInExpr subst@Subst{..} = \case
         (applySubstInExpr subst e)
     ETypeRep t -> ETypeRep
         (applySubstInType subst t)
+    ETypeRepGeneric k t -> ETypeRepGeneric k
+        (applySubstInType subst t)
```

Comment:
Hmm. In theory there should be no type variables in t...

Reply:
We do this sort of substitution in other places though that shouldn't have type variables, though. I just realised how technically unnecessary it is?

Reply:
yeah noticed that while implementing and decided to stick with what we do elsewhere rather than tryting to be clever.